libnmstate/nm/ipv6.py

```diff
@@ -158,3 +163,24 @@ def get_ip_profile(active_connection):
     if remote_conn:
         return remote_conn.get_setting_ip6_config()
     return None
+
+
+def get_route_running():
+    client = nmclient.client()
+    skip_networks = [iplib.IPV6_LINK_LOCAL_NETWORK,
```

Comment:
As we now split the config and running, there is no need to hide any routes.

Reply:
Done.